src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -38,6 +38,7 @@ THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 <#@ Import Namespace="Microsoft.OData.Edm.Vocabularies.V1" #>
 <#@ Import Namespace="Microsoft.OData.Edm.Vocabularies.Community.V1" #>
 <#@ Import Namespace="System.Text"#>
+<#@ Import Namespace="System.Text.RegularExpressions"#>

```

Comment:
please search the security warning to use "RegularExpression " and make sure it's ok to use it.